games-arcade/rocksndiamonds/rocksndiamonds-3.3.1.2.recipe

```diff
@@ -59,5 +62,9 @@ INSTALL()
 	destdir=$appsDir/"Rocks'n'Diamonds"
 	mkdir -p $destdir
 	cp -r rocksndiamonds sounds graphics levels music $destdir
+
```

Comment:
I first tried it by adding "2" in the rdef file, then checked it against FileTypes afterwords to see it worked, then I changed the line according to the previous lines that you handed to us in the past :+1: